hack/cluster-sync.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# ******************************************************
+# Create service account and kubeconfig to access tenant cluster
+# ******************************************************
+./kubevirtci kubectl -n kvcluster create -f ./deploy/infra-cluster-service-account.yaml
+
+export INFRA_KUBECONFIG_IN_TENANT_FILE=_ci-configs/infra_kubeconfig.yaml
+./hack/create-infra-kubeconfig.sh  > $INFRA_KUBECONFIG_IN_TENANT_FILE
+#  Try to find the url in universal way: maybe it is node ip and service port??
+#         sed -i -r 's/127.0.0.1:[0-9]+/192.168.66.101:6443/g' kubeconfig-e2e
+#
+#❯ ./kubevirtci kubectl get node -o wide
+#NAME     STATUS   ROLES                         AGE   VERSION   INTERNAL-IP      EXTERNAL-IP   OS-IMAGE          KERNEL-VERSION          CONTAINER-RUNTIME
+#node01   Ready    control-plane,master,worker   46m   v1.23.6   192.168.66.101   <none>        CentOS Stream 8   4.18.0-383.el8.x86_64   cri-o://1.22.4
+#
+#❯ ./kubevirtci kubectl get svc -o wide -n kvcluster
+#NAME           TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE   SELECTOR
+#kvcluster-lb   ClusterIP   10.101.163.253   <none>        6443/TCP   41m   cluster.x-k8s.io/cluster-name=kvcluster,cluster.x-k8s.io/role=control-plane
+#
+sed -i -r 's/127.0.0.1:[0-9]+/192.168.66.101:6443/g' $INFRA_KUBECONFIG_IN_TENANT_FILE
+
+# ******************************************************
+# Add kubeconfig to secret and create it in tenant
+export INFRA_KUBECONFIG_IN_TENANT_CONTENT=$(cat $INFRA_KUBECONFIG_IN_TENANT_FILE | base64 -w 0)
+envsubst < ./deploy/secret-template.yaml > _ci-configs/tenant_secret.yaml
+
+./kubevirtci kubectl-tenant create -f ./deploy/000-namespace.yaml
+./kubevirtci kubectl-tenant create -f _ci-configs/tenant_secret.yaml
+
+# ******************************************************
+# Apply config
+./kubevirtci kubectl-tenant create -f ./deploy/configmap-template.yaml
+
+# ******************************************************
+# Finally deploy the driver
+# ******************************************************
+# TODO:  The yaml should reference the container image in the local kubevirtci repo (from the image push step)
+
+./kubevirtci kubectl-tenant create -f ./deploy/000-csi-driver.yaml
+./kubevirtci kubectl-tenant create -f ./deploy/020-autorization.yaml
+./kubevirtci kubectl-tenant create -f ./deploy/030-node.yaml
+./kubevirtci kubectl-tenant create -f ./deploy/040-controller.yaml
+
+# ******************************************************
+# Edit storage class
+#- infraStorageClassName: standard
+#+ infraStorageClassName: rook-ceph-block
+
+export INFRA_KUBECONFIG_STORAGECLASS_FILE=_ci-configs/storageclass.yaml
+cp ./deploy/example/storageclass.yaml $INFRA_KUBECONFIG_STORAGECLASS_FILE
+sed -i -r 's/standard/rook-ceph-block/g' $INFRA_KUBECONFIG_STORAGECLASS_FILE
```

Comment:
i can't remember the details here. Does this need to match the storageclass used on the infra cluster? If so, i dont' believe rook-ceph is deployed by kubevirtci anymore. We transitioned to using local storage instead due to issues with resource consumption with rook-ceph.

so maybe use `local` here?

Reply:
You can enable using rook-ceph (if you want) by calling `export KUBEVIRT_STORAGE="rook-ceph-default"` before calling make cluster-up. This should deploy ceph (its part of the default kubevirtci image)

Reply:
yep, we're aware. The issue is resource consumption. We're kind of on the edge (memory wise) due to needing to run two giant VMs nested in the environment to host the guest cluster. Unless we need rook-ceph, local storage would be helpful. We actually removed usage of rook-ceph in the capk repo because of memory issues.

Reply:
I guess local can work, if you want something more dynamic you can try hpp, it doesn't use much memory. But local storage is likely even less memory

Reply:
fyi, this still isn't resolved, which might be causing issues for you

Reply:
did not see issues, I did tests on both ceph and local, both work, it is now updated, will push soon

Reply:
done - updated to local